spec/splitOnJsonata/splitOnJsonata.spec.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const expect = require('chai').expect;
```

Comment:
Did you apply eslint rules?

example of eslint configuration (`.eslintrc.js` file)

```js
module.exports = {
  'extends': 'airbnb-base',
  'env': {
    'mocha': true
  }
};

```

Reply:
It was extending from `eslint:recommended` instead of `airbnb-base`, fixed now